packages/playwright-test/src/transform.ts

```diff
@@ -95,8 +95,10 @@ const builtins = new Set(Module.builtinModules);
 export function resolveHook(isModule: boolean, filename: string, specifier: string): string | undefined {
   if (builtins.has(specifier))
     return;
-  const isTypeScript = filename.endsWith('.ts') || filename.endsWith('.tsx');
-  const tsconfig = isTypeScript ? loadAndValidateTsconfigForFile(filename) : undefined;
+  // In real life, playwright-test is under node_modules, but in the tests it isn't.
+  if (filename.startsWith(kPlaywrightInternalPrefix))
+    return;
+  const tsconfig = loadAndValidateTsconfigForFile(filename);
```

Comment:
I think we should consult `allowJs` property, otherwise this would be surprising?